### ISSUE TITLE
Documentation / Help: Discrepancy between descriptions for "ZZ"/"ZQ" and ":x"/":q"

### ISSUE THREAD

**Issue by GuanyangLuo** [vim/vim#6054]
**Describe the bug**
In "index.txt", the descriptions for "ZZ" and "ZQ" are different from ":x" and ":q". "ZZ" and "ZQ" use the words "store" and "exit", while ":x" and ":q" use the words "write" and "quit". This makes it difficult for beginners like me to discover commands for "write" and "quit", because searching for these keywords will not help me find "ZZ" and "ZQ".

**To Reproduce**
Check "index.txt     For Vim version 8.2.  Last change: 2020 Apr 26"
https://vimhelp.org/index.txt.html#index.txt

**Expected behavior**
Description for "ZZ" should be similar to "write current file if modified, and quit".
Description for "ZQ" should be similar to "quit current file always".

**Comment by brammool** [comment]
> **Describe the bug**
> In "index.txt", the descriptions for "ZZ" and "ZQ" are different from
> ":x" and ":q". "ZZ" and "ZQ" use the words "store" and "exit", while
> ":x" and ":q" use the words "write" and "quit". This makes it difficult
> for beginners like me to discover commands for "write" and "quit",
> because searching for these keywords will not help me find "ZZ" and
> "ZQ".
> 
> **To Reproduce**
> Check "index.txt     For Vim version 8.2.  Last change: 2020 Apr 26"
> https://vimhelp.org/index.txt.html#index.txt
> 
> **Expected behavior**
> Description for "ZZ" should be similar to "write current file if modified, and quit".
> Description for "ZQ" should be similar to "quit current file always".

Yeah, I can see that this can be confusing.  Also, these days it is much
more common to have multiple windows open, thus saying that ZZ closes
the window, and quits when it was the last one, should be more helpful.

Should also explain that a help or preview window doesn't count.  Let me
use the term "edit-window" as a short form to ignore help and preview
windows.

-- 
hundred-and-one symptoms of being an internet addict:
90. Instead of calling you to dinner, your spouse sends e-mail.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
